etc/profile-a-l/singularity.profile

```diff
@@ -0,0 +1,38 @@
+include singularity.local
+include globals.local
+
+noblacklist ${HOME}/.config/singularity
+noblacklist ${HOME}/.local/share/singularity
+
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+
+mkdir ${HOME}/.config/singularity
+mkdir ${HOME}/.local/share/singularity
+whitelist ${HOME}/.config/singularity
+whitelist ${HOME}/.local/share/singularity
+include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+net none
+caps.drop all
+disable-mnt
```

Comment:
Sort disable-mnt down as per template

Reply:
sort.py did not point this out.

Reply:
sort.py does inline sorting of arguments too `private-*`, `seccomp`, ... but no multiline sorting across the file.